### ISSUE TITLE
Training example for StableUnCLIP

### ISSUE THREAD

**Issue by haofanwang** [huggingface/diffusers#2961]
The [pipeline_stable_unclip_img2img.py](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/pipeline_stable_unclip_img2img.py) is great, but how do I finetune the model just as done in [train_text_to_image.py](https://github.com/huggingface/diffusers/blob/main/examples/text_to_image/train_text_to_image.py). For now, I achieve it on my own, but the loss doesn't decrease as expected. There seems to be of some bugs. @patil-suraj Can you PR a training example for it?

**Comment by patrickvonplaten** [comment]
Hmm yeah we haven't looked too much into fine-tuning here to be honest and I'm not sure if we find the time to do so.

**Comment by haofanwang** [comment]
It would be very appreciated if pseudocode for the [core part](https://github.com/huggingface/diffusers/blob/main/src/diffusers/pipelines/stable_diffusion/pipeline_stable_unclip_img2img.py#L759) (how the image embedding is added in the forward process) can be provided. Then I can implement the full training code. @patrickvonplaten

**Comment by eeyrw** [comment]
I am quite curious about training cascade super resolution module of unCLIP implementation. The current community solution is use SD with CLIP image embedding version to generate image which is not good.

**Comment by haofanwang** [comment]
Which community implementation do you use? @eeyrw

**Comment by eeyrw** [comment]
This one: https://huggingface.co/docs/diffusers/v0.14.0/en/api/pipelines/stable_diffusion/image_variation#diffusers.StableDiffusionImageVariationPipeline

**Comment by 00why00** [comment]
Same issue. Have you solved it? @haofanwang 
How the image embedding is added in the forward process

**Comment by chenrxi** [comment]
Same issue. Have you solved it? @haofanwang. Many thanks

**Comment by mynameischaos** [comment]
> Same issue. Have you solved it? @haofanwang How the image embedding is added in the forward process

Same issue
